package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sonarlint-vscode",
   "displayName": "SonarLint",
   "description":
-    "On-the-fly bug and quality issues detection for JavaScript, Python and PHP",
+    "On-the-fly bug and quality issues detection for JavaScript, TypeScript, Python and PHP",
```

Comment:
@henryju should it be in different commit?

Reply:
@vilchik-elena Indeed